nuxt.config.ts

```diff
@@ -155,7 +155,7 @@ const config: NuxtConfig = {
       { code: 'en', iso: 'en-US', name: 'English', file: 'en-US.json' },
       { code: 'es', iso: 'es-ES', name: 'Español (España)', file: 'es.json' },
       { code: 'fr', iso: 'fr-FR', name: 'Français', file: 'fr-FR.json' },
-      { code: 'nb', iso: 'nb-NO', name: 'Bokmål', file: 'nb_NO.json' },
+      { code: 'nb', iso: 'nb-NO', name: 'Norsk', file: 'nb_NO.json' },
```

Comment:
```suggestion
      { code: 'nb', iso: 'nb-NO', name: 'Norsk bokmål', file: 'nb_NO.json' },
```

This is the proper name for it. As there are two variants of "Norsk", we should explicitly mention which one it is.